design-documents/graph-ql/coverage/gift-wrapping.md

```diff
@@ -176,17 +196,27 @@ type CartItemUpdateInput {
     gift_wrapping_id: ID @doc(description: "The unique identifier of the gift wrapping to be used for the cart item")
     gift_message: GiftMessageInput @doc(description: "Gift message details for the cart item")
 }
+type RemoveItemFromCartInput {
```

Comment:
Just needs to switch from `type` to `input`

Reply:
yes, you are right. I will change that.